src/gentropy/study_validation.py

```diff
@@ -62,8 +62,8 @@ def __init__(
 
         # Running validation:
         study_index_with_qc = (
-            study_index.validate_unique_study_id()  # Flagging duplicated study ids
-            .validate_study_type()  # Flagging non-supported study types.
+            study_index.deconvolute_studies()  # Deconvolute studies where the same study is ingested from multiple sources
```

Comment:
I can see, why it can be challenging to merge, them as the index is already merged :/ Mayby this reasoning should be in the validation ?